evoxbench/test_suites/in1kmop.py

```diff
@@ -31,5 +31,10 @@ def in1kmop(problem_id):
     elif problem_id == 9:
         return MobileNetV3Benchmark(
             objs='err&params&flops&latency', normalized_objectives=False)
+    elif problem_id == 10:
```

Comment:
The **MoSegNASBenchmark** and ImageNet are not related, please add a new test suite file.

Reply:
I see, thanks for correction!